packages/memory/src/bus/lib/redis.test.ts

```diff
@@ -91,7 +95,7 @@ describe('constructor', () => {
 	});
 
 	test('Subscribes to messageBuffers in sub redis', () => {
-		expect(bus['sub'].on).toHaveBeenCalledWith('messageBuffer', bus['messageBufferHandler']);
+		expect(bus['sub'].on).toHaveBeenCalledWith('messageBuffer', expect.any(Function));
```

Comment:
This was secretly the only reason I went with `, this.handler` instead of `, (val) => this.handler(val)`. Make the test coverage more reliable 🙂